multihost_test/bz_automation/test_chkpwd-onlyroot.py

```diff
@@ -60,3 +60,16 @@ def test_bz675835(self, multihost, create_localusers):
             exceute_cmd(multihost, "sh /tmp/bz675835.sh local_anuj x")
         with pytest.raises(subprocess.CalledProcessError):
             exceute_cmd(multihost, "sh /tmp/bz675835.sh pamtest1 x")
+
+    def test_1949137(self, multihost, create_localusers, create_system_user):
+        """
+        :title: pam_usertype has flawed logic for system accounts.
+        :id: fe3e1048-9483-11ec-b9bb-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1949137
+        """
+        file_location1 = "/multihost_test/bz_automation/script/system-auth"
+        multihost.client[0].transport.put_file(os.getcwd() +
+                                               file_location1,
+                                               '/etc/pam.d/system-auth')
```

Comment:
Can you create another generic function to handle the backup and restore of the PAM service files? The input could be a list of the files.

Reply:
This one is not about backup and restore , its the new pam config we are sending to client server  , the original file backup and restore will be taken care by fixture "create_localusers"

Reply:
Sorry, I misunderstood what you were doing. Now that I got it I have one new question. Why are you hiding the backup and restore of the configuration files inside the `create_localusers()` function? I know you didn't change this part of the code in this PR, but wouldn't it be better to have that separated in a separate function?

Reply:
changes are done